### PR TITLE
K8s update doc 1010

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/kubernetes/pages/quickstart-with-aks.adoc [hunk 1]
```diff
@@ -10,23 +11,35 @@ This is a quickstart guide for deploying TigerGraph single servers and clusters
...
+*Step 1: Generate deployment manifest*.
+Clone the https://github.com/tigergraph/ecosys.git[TigerGraph ecosystems repository] and change into the `k8s` directory.
+You can edit the `kustimization.yaml` file in the `aks` folder to change the namespace and image name for your deployment.
+The default namespace is `default`, and the default image is the official docker image for TigerGraph in the latest.
+No need to edit the files if no changes are needed.
 
-Next, run the `./tg` script in the `k8s` directory to generate the deployment manifest for a single-server deployment. A `deploy` directory will be created automatically and you should find the manifest named `tigergraph-aks.yaml` in the directory.
+Next, run the `./tg` script in the `k8s` directory to generate the deployment manifest for a single-server deployment.
+A `deploy` directory will be created automatically, and you should find the manifest named `tigergraph-aks.yaml` in the directory.
```
Comment (hanruytg) [inline modules/kubernetes/pages/quickstart-with-aks.adoc:34]: Change to `tigergraph-aks-[deployment].yaml` (same goes for eks and gke)

File: modules/kubernetes/pages/quickstart-with-gke.adoc [hunk 2]
```diff
@@ -31,7 +42,7 @@ $ ./tg gke kustomize -s 1
 
 [source,console]
 ----
-$ kubectl apply -f deployment/tigergraph-gke.yaml
+$ kubectl apply -f deploy/tigergraph-gke.yaml
```
Comment (hanruytg) [inline modules/kubernetes/pages/quickstart-with-gke.adoc:45]: I noticed this part isnt present for eks and aks?

**Review by hanruytg — CHANGES_REQUESTED** [review]
As part of the delete namespace bugfix, the tg script will now create files at `deploy/tigergraph-gke-[namespace].yaml` instead of `deploy/tigergraph-gke.yaml` (same for aks/eks), can you inform users of this change and update the existing references? Thanks

File: modules/kubernetes/pages/quickstart-with-gke.adoc (hunk 2 above)
Comment (lennessyy) [reply modules/kubernetes/pages/quickstart-with-gke.adoc]: They should be at around line 40 - 45 for both documents

File: modules/kubernetes/pages/quickstart-with-aks.adoc (hunk 1 above)
Comment (lennessyy) [reply modules/kubernetes/pages/quickstart-with-aks.adoc:36]: I added a note that the actual filename will contain the namespaces provided by the user.

**Review by hanruytg — DISMISSED** [review]

**Review by hanruytg — APPROVED** [review]

**Pull request merged**